### PR TITLE
/admin-view/students/{studentID}

### PR DESCRIPTION
**Pull request by SaurabGhimire**

### CODE REVIEW

File: src/main/java/com/cs544/project/controller/AdminViewController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/admin-view")
+public class AdminViewController {
```
Comment (bibangamba) [inline src/main/java/com/cs544/project/controller/AdminViewController.java:15]: same comment about view-based controllers vs resource-based controllers

File: src/main/java/com/cs544/project/controller/AdminViewController.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @Autowired
+    StudentService studentService;
+
+    @GetMapping("/students/{studentId}")
+    public ResponseEntity<?> getStudentsByStudentID(@PathVariable("studentId") String studentId) throws CustomNotFoundException {
+        Student student = studentService.getStudentByStudentID(studentId);
```
Comment (bibangamba) [inline src/main/java/com/cs544/project/controller/AdminViewController.java:21]: what happens if no student exists with that id? no exception?

Comment (bibangamba) [reply src/main/java/com/cs544/project/controller/AdminViewController.java:21]: i've seen that it comes as optional,  should check for null and send 404

Comment (SaurabGhimire) [reply src/main/java/com/cs544/project/controller/AdminViewController.java:21]: The exception is handle by AOP, CustomExceptionHandler class

**Pull request merged**